syntax = "proto3";

package employee;

message Response {
    int32 StatusCode = 1;
    string ResponseMessage = 2;
}